src/app/(admin)/admin/page.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { render } from "../../../test/utils";
+import { render } from "~/test/utils";
```

Comment:
why did you work on admin?

Reply:
i didn't work on that , i thing it was lint:fix that did that. The last merge u did before i pulled had lint and test issues but now they are all pinned on me.  while the only thing i did was what was asked of me